### ISSUE TITLE
importlib.metadata not available in python 3.6

### ISSUE THREAD

**Issue by nabecker2** [IBM/tnz#20]
Running into this issue on python 3.6. Apparently importlib.metadata was added in python 3.8.

```
  ...
  File "/home/nick/.local/lib/python3.6/site-packages/tnz/ati.py", line 3657, in set
    return ati.set(name, value, xtern=xtern, trace=trace)
  File "/home/nick/.local/lib/python3.6/site-packages/tnz/ati.py", line 1663, in set
    self.display = value
  File "/home/nick/.local/lib/python3.6/site-packages/tnz/ati.py", line 2979, in display
    from . import zti
  File "/home/nick/.local/lib/python3.6/site-packages/tnz/zti.py", line 51, in <module>
    from importlib.metadata import entry_points
ModuleNotFoundError: No module named 'importlib.metadata'
```

**Comment by najohnsn** [comment]
Thank you!

Not sure if it's what you expect... but my first reaction is to bump up the requirement to Python 3.8.

**Comment by nabecker2** [comment]
Given that the tnz documentation says that python 3.6 is the minimum required version to use the package, it's not quite what I expected.

Requiring python 3.8 would be one option, but I did some digging and it looks like they created a compatibility module called importlib_metadata for python releases where importlib.metadata isn't available. I created a pull request (#21) for a possible workaround that would fallback to the compatibility module if importlib.metadata fails to import.  I tested it using python 3.6 and it seems to resolve the issue.  Let me know what you think.
